### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by simonhorner** [codewars/codewars.com#2974] [Q&A] [answered]
I opened a few years ago then didn't do anything for about 4 years until a couple of kata recently. I started a new job and opened a new Code Wars account. I copied my previous solutions over from my old account but now my account is locked. I can understand not copying other people's solutions but didn't realise copying my own would cause my account to be locked.

Any chance my account could be unlocked?

My username for the old account is shorner. The new account is simonhorner.

**Comment by Blind4Basics** [comment, marked as answer]
handled on discord.
